binding/rust/README.md

```diff
@@ -0,0 +1,51 @@
+# Cobra Voice Activity Detection engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Cobra is a highly accurate and lightweight voice activity detection (VAD) engine.
+
+## Compatibility
+
+- Rust 1.54+
+- Runs on Linux (x86_64), macOS (x86_64), Windows (x86_64), Raspberry Pi, NVIDIA Jetson (Nano), and BeagleBone
+
+## Installation
+First you will need [Rust and Cargo](https://rustup.rs/) installed on your system.
+
+To add the cobra library into your app, add `pv_cobra` to your apps `Cargo.toml` manifest:
+```toml
+[dependencies]
+pv_cobra = "*"
+```
+
+## Usage
+
+Create an instance of the engine:
+
+```rust
+use cobra::Cobra;
+
+let app_id = "..."; // AppID provided by Picovoice Console (https://picovoice.ai/console/)
```

Comment:
should we use access_key instead now?

Reply:
Maybe, I already had it all as App ID and I figured it could get done as part of converting everything else in one go. What do you think?

Reply:
not sure. I guess you can choose how/when to do it, eventually they are going to be changed.